packages/components/src/icons/blaze-colored.tsx

```diff
@@ -0,0 +1,27 @@
+import { Defs, LinearGradient, Path, Stop, SVG } from '@wordpress/primitives';
+
+const icon = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
+			d="M87.7351 112.095C105.444 103.34 117.622 85.0917 117.622 63.9995C117.622 34.3851 93.6144 10.3779 64 10.3779C34.3856 10.3779 10.3784 34.3851 10.3784 63.9995C10.3784 83.4624 20.7477 100.503 36.2631 109.899C27.9789 103.223 24.8921 92.8046 24.8921 81.7366C24.8921 61.4956 43.6643 47.6298 43.6643 47.6298C43.6643 47.6298 45.4641 59.955 54.6147 59.955C71.8225 59.955 65.5651 32.2234 65.5651 32.2234C65.5651 32.2234 81.2086 41.4672 85.9016 66.1175C92.159 63.0362 93.7233 53.7924 93.7233 53.7924C93.7233 53.7924 103.109 64.5769 103.109 81.524C103.109 98.5452 95.2162 107.464 87.7351 112.095ZM75.2325 116.443C77.5248 114.522 85.1838 107.43 85.1838 98.4229C85.1838 81.4758 72.2981 72.2801 72.2981 72.2801C72.2981 72.2801 70.7337 89.2272 60.8721 89.2272C54.6147 89.2272 51.486 84.6053 51.486 84.6053C51.486 84.6053 42.8161 89.2272 43.6643 99.8059C45.2538 110.278 52.5769 115.632 54.2663 116.74C57.4229 117.319 60.6762 117.621 64 117.621C67.8526 117.621 71.6102 117.215 75.2325 116.443Z"
+			fill="url(#paint-blaze-icon-linear-gradient)"
+		/>
+		<Defs>
+			<LinearGradient
+				id="paint-blaze-icon-linear-gradient"
```

Comment:
🤦‍♂️ I see. I'll omit the colored variant of the icon in this PR since [applying linear gradients on SVGs appears to be more involved](https://fvsch.com/svg-gradient-fill) than I expected.